ntbs-ui-tests/Steps/Steps.cs

```diff
@@ -9,18 +9,31 @@
 
 namespace ntbs_ui_tests.StepDefinitions
 {
+    public class StepsData
+    {
+        public int NotificationId;
+    }
+    
     [Binding]
     public class Steps
     {
         private readonly IWebDriver Browser;
         private readonly SeleniumServerFactory<Startup> Server;
 
+        private readonly StepsData stepsData = new StepsData();
+
         public Steps(IWebDriver driver, SeleniumServerFactory<Startup> server)
         {
             Browser = driver;
             Server = server;
         }
 
+        [Given(@"I am on current notification page")]
+        public void GivenIAmOnNotificationPage()

```

Comment:
We're referring to this page as `overview` elsewhere - think it'll probably pose more readable down the line with a rename